README.md

```diff
@@ -66,13 +67,27 @@ Each link in this category is scoped to posts tagged with Archivematica.
 - 2016-02  Mitcham J (University of York), Awre C, Allinson A, Green R, Wilson S, (University of Hull). "Filling the Digital Preservation Gap - A Jisc Research Data Spring project: Phase Two report" [report](resources/2016-02-01-JiscDataSpringFinalReportphase2.pdf).
 - 2016 Bronwen Sprout and Sarah Romkey: "IMPLEMENTATION: Building a Preservation Strategy around Archivematica" [book chapter](https://books.google.ca/books/about/Building_Trustworthy_Digital_Repositorie.html?id=I0V5DAAAQBAJ&printsec=frontcover&source=kp_read_button&redir_esc=y#v=onepage&q&f=false).
 
-## 2015 and prior
+## 2015
 
 - 2015-03-02 Brad Houston, University of Wisconsin-Milwaukee: The American Archivist Review: Archivematica [article](https://reviews.americanarchivist.org/2016/07/02/archivematica/).
-- 2015-01 Bronwen Sprout and Mark Jordan: Archivematica As a Service: COPPUL’s Shared Digital Preservation Platform [article](resource/archivematica-as-a-service-coppul.pdf).
+- 2015-01 Bronwen Sprout and Mark Jordan: Archivematica As a Service: COPPUL’s Shared Digital Preservation Platform [article](resources/archivematica-as-a-service-coppul.pdf).
+
+## 2014
+
 - 2014-10-21 Corey Davis (University of Victoria Libraries): "Archiving the Web: A Case Study from the University of Victoria" [code4lib journal article](https://journal.code4lib.org/articles/10015).
 - 2014 Solomon R. Guggenheim Foundation Electronic Records Management Start-up Project: [document](resources/guggenheim-library-archives-preservation-and-access-formats-2014.pdf).
 
+## 2013
+
+- 2013-09: Van Garderen, P. Mumma, C. at iPres 2013: "Realizing the Archivematica vision: delivering a comprehensive and free OAIS implementation." [article](resources/2013-09-02-Realizing-the-Arhivematica-Vision-MummaVanGarderen-iPres.pdf).
+
+## 2012
+
+- 2012-10: Van Gardered, P. Mumma, C. at iPres 2012: "The Community driven Evolution of the Archivematica Project." [article](resources/2012-10-iPres-Paper49-Community-driven-evolution-of-Archivematica-project-VanGarderen-Mumma.pdf).
```

Comment:
Should read "Van Garderen", yeah?

Reply:
Yowzerz! These are bad typos, even for me! Thanks Tim1